components/scream/cime_config/buildnml

```diff
@@ -675,13 +671,13 @@ def create_input_files(caseroot, screamroot, rundir):
         generated_files = get_child(tree.getroot(),"generated_files")
 
     result = {}
-    for file in generated_files:
-        expect("name" in file.attrib, "file element missing required 'name' attribute")
-        expect("format" in file.attrib, "file element missing required 'format' attribute")
-        filename    = os.path.join(rundir, file.attrib["name"])
-        file_format = file.attrib["format"]
+    for file_ in generated_files:
```

Comment:
Out of curiosity, why this change?

Reply:
`file` is a special name in python that usually refers to the class that handles files. The language allows you to override names like this, but it's considered poor practice and pylint flagged it.

Reply:
Ah, I was wondering if that was the case. I tried to open a python file and type `for file in blah` but my vim did not highlight `file`, so I thought that wasn't the case. Good to know. Thanks.